_posts/2020-01-07-jump_dev_2020.md

```diff
@@ -16,31 +16,22 @@ with interests in the software aspects of [JuMP](https://github.com/JuliaOpt/JuM
 related packages. The workshop is open for anybody to attend. In particular, we invite new
 contributors and those who have not met the core development team.
 
+## COVID-19
+
+At present, we are still planning for JuMP-dev to proceed as usual. However, we
+are aware that some institutions have placed travel bans until mid- to late-May.
+
+If these restrictions are extended, or the situation in Belgium changes, we will
```

Comment:
While it's too early to make a call, there are plausible scenarios in which we may have to cancel the physical event. Therefore, we recommend ...

Note: By saying physical event I'm leaving the door open for an alternative.